src/zenml/cli/utils.py

```diff
@@ -864,7 +864,7 @@ def get_stack_by_id_or_name_or_prefix(
         is_shared=True,
     )
 
-    named_stacks = user_only_stacks + shared_stacks
+    named_stacks = user_only_stacks + shared_stacks  # type: ignore[operator]
```

Comment:
As a general strategy, please try to not silence mypy this way. Instead, add some `assert isinstance(...)` statements after `list_stacks` calls